lib/publiccloud/instance.pm

```diff
@@ -619,23 +621,25 @@ sub measure_boottime() {
 
 Save data collected with measure_boottime in a DB;
 Mainly stored on a remote InfluxDB on a Grafana server.
-
+To activate boottime push, shall be available results and
+  PUBLIC_CLOUD_PERF_PUSH_DATA true/not 0 and
+  _SECRET_PUBLIC_CLOUD_PERF_DB_TOKEN and PUBLIC_CLOUD_PERF_DB_URI defined
 =cut
 
 sub store_boottime_db() {
     my ($self, $results) = @_;
-    return unless (get_var('PUBLIC_CLOUD_PERF_PUSH_DATA') && $results);
-
-    my $url = get_var('PUBLIC_CLOUD_PERF_DB_URI');
+    my $data_push = get_var('PUBLIC_CLOUD_PERF_PUSH_DATA', 1);
```

Comment:
I'd say the old style was better because it immediately returns if not enabled.

```suggestion
    return unless (get_var('PUBLIC_CLOUD_PERF_PUSH_DATA', 1) && $results);
```

Plus, you could replace the next following `get_var` calls with `get_required_var` - if the data push is enabled, the performance database variables should be treated as required variables IMHO.

Reply:
same comment as above for data collect

Reply:
> Plus, you could replace the next following get_var calls with get_required_var - if the data push is enabled, the performance database variables should be treated as required variables IMHO.

the whole point of this PR is to remove need of defining this variables elsewhere . Maurizio did everything as expected

Reply:
get_required_var would cause test to die, unwanted here

Reply:
and also `get_required_var` not allow to provide default value which major idea of this PR

Reply:
Also here: I don't understand, but as this is just a style suggestion, just ignore it.